test/integration_test.py

```diff
@@ -227,8 +227,10 @@ def _wait_for_indexer():
 
         if index and delete:
             for catalog in catalogs:
+                notifications = catalog.notifications_with_duplicates()
+                notifications = notifications[-(len(notifications) // 8):]
```

Comment:
Why the reversal (negative start index)? Can you just drop a single notification to the same effect?

Reply:
A single notification has a higher likelihood of being in the set of the duplicates, a larger amount of notifications had more chances of not sharing a notifications with the duplicates. I should simple chose the notifications without duplicates and remove one from there, that way it is simpler. 
The reversal was intended to remove from the start of the list, rather than the end were the duplicate notifications get appended. But I realize now that the `-` wasn't necessary.